docs/samples/object-storage.md

```diff
@@ -0,0 +1,70 @@
+# S3 Object Storage with vSphere Volumes
+
+[Minio](https://www.minio.io/) is lightweight AWS S3 compatible object storage server built for cloud applications.
+
+vSphere Docker Volume Service enables running stateful containers backed by proven enterprise class storage. 
```

Comment:
Nit : We should introduce vDVS acronym here which is used in line 7 - "vSphere Docker Volume Service(vDVS) enables ..."